packages/webview_flutter/webview_flutter_wkwebview/lib/src/webkit_webview_controller.dart

```diff
@@ -630,11 +630,34 @@ class WebKitWebViewController extends PlatformWebViewController {
   }
 
   Future<void> _injectConsoleOverride() {
+    // Using the replacer parameter of JSON.stringify() to solve the error
+    // TypeError: JSON.stringify cannot serialize cyclic structures.
+    // See https://github.com/flutter/flutter/issues/144535.
+    //
+    // Considering this is just looking at the logs printed via console.log,
+    // the cyclic object is not important, so remove it.
     const WKUserScript overrideScript = WKUserScript(
       '''
+function removeCyclicObject() {
```

Comment:
This should have a name that cannot plausibly cause collisions on actual web pages, since it is injected into the page's context. E.g., maybe a Flutter prefix and a dynamically generated UUID? @ditman Any suggestions on best practice for avoiding naming collisions in JS?

Reply:
@stuartmorgan At the expense of some performance this function could be defined inside of the `log` function that calls it (line 659)... But if we're not worried about a function `log` colliding with the functions of a webpage, I wouldn't worry too much about `removeCyclicObject` either :)

(In the engine we normally prefix with `_` the name of the function so there's a fewer change of a global collision, but there's no guarantee that it won't collide. For a UUID you'd have to strip the `-` characters for it to be a valid function name, but it could definitely work)

Reply:
> We wouldn't need to do it in JS; we'd generate it in Dart, and then string interpolate that into the string we inject.

Ah you're right, it's "free" then. `_flutter_log_override` sounds like a good name. You can even have a small `_flutter_webview_plugin_overrides` "namespace" object that contains all the methods that you're injecting, so it's easy to hide/move them around.

Reply:
Done.